README.md

```diff
@@ -30,7 +30,7 @@ you could lint your package like this:
 
     $ dartanalyzer --options analysis_options.yaml .
     
-and see any violations of the `annotate_overrides`, `hash_and_equals`, and `prefer_is_not_empty` rules in the console.  In practice you would probably configure quite a few more rules (the full list is [here][lints]).
+and see any violations of the `annotate_overrides`, `hash_and_equals`, and `prefer_is_not_empty` rules in the console.  In practice you would probably configure quite a few more rules (the full list is [here][lints]).  The evolving list of lints enforced internally at Google may serve as inspiration [here][google-lints].

```

Comment:
Might be just my personal preference but I dislike links with text `here`.  
Better would be a text that tells what's behind the link.
Why not `... rules ([the full list][lints])`

Reply:
I totally agree.  Thanks!

Reply:
My writing style is somewhat different, so the tone might not match, but I'd write something like this:
```
... more rules.

To help you choose the rules you want to enable for your package, we have provided both a [complete list of rules][lints] and a list of the [lints that are enforced internally at Google][google-lints].
```

Reply:
perfectly fine for me as well

Reply:
Updated to @bwilkerson's suggestion. 👍